plugins/Source.py

```diff
@@ -51,14 +51,14 @@ def init(self, logger):
 `source=*`.'''))
 
     def check(self, tags):
-        if u"source" not in tags and u"ref" not in tags:
+        if "source" not in tags and "ref" not in tags:
             return
 
-        for tag in (u"source", u"ref"):
+        for tag in ("source", "ref"):
             if tag in tags:
                 value = tags[tag].lower()
-                if u"google" in value:
-                    return {"class": 706, "subclass": 2, "text": {"en": u"Google {0}={1}".format(tag, value)}}
+                if any("google" in v and not v.endswith("buildings") for v in value.split(';')):
```

Comment:
Maybe add a `.strip()`?

Reply:
Sure, done :) .
According to TagInfo there's no cases of `source=google_africa_buildings ; something`, thus with a space between `buildings` and `;` but it's more future proof indeed.